### PR TITLE
wp-now: avoid stopping wp-now for lower node versions

### LINKED ISSUE #3: Local Environment: Don't exit if Node.js version < 18
## Description

According to @luisherranz, wp-now works well on Node.js version 16. However, this line prevents people from using it there:

https://github.com/WordPress/wordpress-playground/blob/79272d3bc2e64db37a5ea2d948b67556ee4b07b0/packages/wp-now/src/main.ts#L3-L12

Furthermore, that binary may be executed using a runtimes different than Node.js.

## Proposed solution

* Don't exit, only issue a warning about the recommended runtime 
* Don't assume `parseInt(process.versions.node.split('.')[0]);` is set 

cc @wojtekn @danielbachhuber @sejas

### PR DESCRIPTION
**Pull request by sejas**
- Closes https://github.com/WordPress/playground-tools/issues/3

### What?

Show a warning message and avoid stopping `wp-now`.

### Why?

- Asked here https://github.com/WordPress/playground-tools/issues/3
- I tested it with node 16, and wp-now works.

### Testing instructions

- Run `nvm use`
- `npm install`
- `npx nx build wp-now`
- `node dist/packages/wp-now/main.js start`
- Observe it works with node 18 without any warning
- Change your current node version: `nvm install 16.0.0`
- `nvm use 16.0.0`
- Run: `node dist/packages/wp-now/main.js start`
- Observe a warning: `You are running Node.js version 16, but this application requires at least Node.js 18. Please upgrade your Node.js version.`
- Observe that wp-now has successfully started.

### CODE REVIEW

File: packages/wp-now/src/main.ts [hunk 1]
```diff
@@ -2,13 +2,12 @@ import { runCli } from './run-cli';
 
 const requiredMajorVersion = 18;
 
-const currentNodeVersion = parseInt(process.versions.node.split('.')[0]);
+const currentNodeVersion = parseInt(process.versions?.node?.split('.')?.[0]);
 
 if (currentNodeVersion < requiredMajorVersion) {
-	console.error(
+	console.warn(
 		`You are running Node.js version ${currentNodeVersion}, but this application requires at least Node.js ${requiredMajorVersion}. Please upgrade your Node.js version.`
```
Comment (danielbachhuber) [inline packages/wp-now/src/main.ts]:
```suggestion
		`You are running Node.js version ${currentNodeVersion}, but this application recommends at least Node.js ${requiredMajorVersion} and isn't guaranteed to work on lower versions. Please upgrade your Node.js version.`
```

**Review by danielbachhuber — APPROVED** [review]
Works as expected! Thanks for the detailed test instructions 😊

File: packages/wp-now/src/main.ts (hunk 1 above)
Comment (sejas) [reply packages/wp-now/src/main.ts]: Thank you! Updated: https://github.com/WordPress/playground-tools/pull/37/commits/cbe9b808d62bc4087815404bf54d35949478ebb8

Comment (sejas) [reply packages/wp-now/src/main.ts]: Thank you! Updated: https://github.com/WordPress/playground-tools/pull/37/commits/cbe9b808d62bc4087815404bf54d35949478ebb8

**Comment by adamziel** [comment]
cc @luisherranz

**Pull request merged**